.github/workflows/linters.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           npm install --save-dev hint@6.x
           [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
       - name: Webhint Report
```

Comment:
- [ ] Since this is a javascript project, kindly use [javascript linter configuration](https://github.com/microverseinc/linters-config/tree/master/javascript), it will solve your errors